nixio/test/test_nix_compatibility.py

```diff
@@ -20,8 +20,12 @@
 BINDIR = tempfile.mkdtemp(prefix="nixpy-tests-")
 
 # skip these tests if nix isn't available
-pytestmark = pytest.mark.skipif("skip()",
-                                reason="Compatibility tests require NIX")
+if pytest.config.getoption("--force-compat"):
+    print("Forcing compatibility tests")
+    maketests(BINDIR)
+else:
+    pytestmark = pytest.mark.skipif("skip()",
+                                    reason="Compatibility tests require NIX")
 
```

Comment:
tiny thing that crossed my mind: is the message self-explanatory? Maybe something like " require nix C++ library"?

Reply:
Sure, I could change it, though it's not really important. The compatibility tests aren't meant for anyone but us. I'll change it anyway, though.